be/src/exec/exec_node.h

```diff
@@ -46,6 +45,11 @@ class TupleRow;
 class DataSink;
 class MemTracker;
 
+namespace vectorized {
+class Block;
```

Comment:
Where does class ```Block``` come from?

Reply:
Just keep the interface for compile be code do not contain the real code.